### PR TITLE
ReklamStore Bid Adapter

### PR DESCRIPTION
**Pull request by ReklamStoreIT**
ReklamStore Bid Adapter

## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by bretg** [comment]
Docs PR at https://github.com/prebid/prebid.github.io/pull/1205

File: modules/reklamstoreBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'reklamstore';
+const ENDPOINT_URL = '//ads.rekmob.com/m/prebid';
+const CURRENCY = 'USD';
+const TIME_TO_LIVE = 360;
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['reklamstore'],
```
Comment (jaiminpanchal27) [inline modules/reklamstoreBidAdapter.js]: No need to add aliases if you are repeating bidder_code here. aliases are to be used if you want to use some short code for bidder in adUnit config.

File: modules/reklamstoreBidAdapter.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+  },
+  /**
+  * Make a server request from the list of BidRequests.
+  *
+  * @param {validBidRequests[]} - an array of bids
+  * @return ServerRequest Info describing the request to the server.
+  */
+  buildRequests: function (validBidRequests) {
+    let requests = [];
+    let loc = utils.getTopWindowLocation();
```
Comment (jaiminpanchal27) [inline modules/reklamstoreBidAdapter.js]: Use http://prebid.org/dev-docs/bidder-adaptor.html#referrers 
`utils.getTopWindowLocation();`  is a deprecated function and will be removed soon

File: modules/reklamstoreBidAdapter.js [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+      }
+
+      return bidResponses;
+    } catch (err) {
+      utils.logError(err);
+      return [];
+    }
+  },
+  /**
+     * Register the user sync pixels which should be dropped after the auction.
```
Comment (jaiminpanchal27) [inline modules/reklamstoreBidAdapter.js]: Please fix spacing in all the jsdoc

File: modules/reklamstoreBidAdapter.js [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+      });
+    });
+
+    return syncs;
+  }
+}
+registerBidder(spec);
+
+function getDeviceType() {
+  var PHONE = 0;
```
Comment (jaiminpanchal27) [inline modules/reklamstoreBidAdapter.js]: Use let/const in place of var

File: test/spec/modules/reklamstoreBidAdapter_spec.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai';
+import { spec } from 'modules/reklamstoreBidAdapter';
+
+describe('reklamstoreBidAdapterTests', function() {
```
Comment (jaiminpanchal27) [inline test/spec/modules/reklamstoreBidAdapter_spec.js]: Add unit test for user sync

**Comment by ReklamStoreIT** [comment]
Changes are done.

**Pull request merged**